app/scripts/controller-init/messengers/smart-transactions-controller-messenger.ts

```diff
@@ -1,4 +1,10 @@
 import { Messenger } from '@metamask/base-controller';
+import {
```

Comment:
Nit: Since these are all types, we can make this a type import

```suggestion
import type {
```